dashboard/src/api/TreeDetails.tsx

```diff
@@ -244,15 +244,18 @@ const fetchTreeCommitHistory = async (
   origin: string,
   gitUrl: string,
   gitBranch: string,
+  filters: TTreeDetailsFilter,
 ): Promise<TTreeCommitHistoryResponse> => {
+  const searchParam = mapFiltersToUrlSearchParams(filters);
```

Comment:
also, can you add a `TSDOC`  `/** @deprecated */` to the function so we stop the bleeding